sample/Messaging.Sample.Kafka/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Messaging.Sample.Kafka
+{
+    public class Program
+    {
+        static void Main()
+        {
+            using (var app = new SomeApp(s =>
+                s.AddMessaging(builder =>
+                {
+                    builder.AddJson();
+                    builder.AddKafka(o =>
+                    {
+                        o.Properties.BrokerList = "localhost:9092";
+                        o.Properties.GroupId = "Sample-GroupId";
+                        o.Properties.Add("socket.timeout.ms", 1000); // sample unmapped setting
+                        o.Subscriber.ConsumerCreatedCallback =
+                            consumer => consumer.OnError += (sender, error)
+                                => Console.WriteLine($"Consumer error: ${error}");
+
+                    });
+                    builder.ConfigureOptions(o =>
+                    {
+                        o.DiscoveryOptions.MessageHandlerAssemblies.Add(typeof(SomeMessage).Assembly);
+                        o.DiscoveryOptions.IncludeNonPubicHandlers = true;
+                    });
+                })))
+            {
+
+                app.Run();
```

Comment:
I guess this is effectively your "integration test" at the moment? Maybe it could be made an explicit integration test that could be run by CI for each commit?

Reply:
Yes. I wasn't planning to use this as an integration test at CI. Rather actually writing tests :)
I'm considering this as a draft to discuss the design before we carry on adding tests and CI, etc.

Reply:
Understood :-)